src/js/lang.js

```diff
@@ -0,0 +1,154 @@
+    'section2-pantografy': 'Pantographs',
+    'section2-realizujemy': 'We perform repairs of current collectors for electrical traction units, including all types used in Poland and abroad.',
+    'section-produkcja': 'Production',
+    'section3-cewki': 'Stator winding and rotors <br class="d-none d-lg-inline">for AC/DC machines',
+    'section3-nasze': 'Our facilities allow production of rotor and stator coils for any type of machine, based on documentation delivered by the client, as well as reproduction of winding upon delivery of the damaged rotor or stator.',
+    'section4-metalowe': 'Metal spare parts for electrical machines',
+    'section4-zapewniamy': 'When repairing motors, we can produce and replace metal parts (rotor shafts, covers, endshields, labyrinths etc.)',
+    'section4-mamy': 'As with the coils, we are able to start production of any metal spare part, based on documentation, as well as to reproduce them upon delivery of the parts by a client.',
+    'about-firma': 'About us',
+    'about-poprzez': 'Focusing on the selected submarket, we gained expertise in traction unit repairs',
+    'about-prezes': 'President of the board',
+    'about-wiedza': 'Know-how',
+    'about-efektywna': 'Practical know-how on construction and exploitation of electrical machines is the base for operation and development of our company.',
+    'about-przez': 'Over years of experience acquired during the company operation, we have developed solutions for traction unit service, allowing us to effectively pursue the essential technical goals.',
+    'section-doswiadczenie': 'Experience',
+    'doswiadczenie-profesjonalizm': 'Our professional approach and excellent technical achievements regarding traction unit repairs exceed our client’s expectations and allow us to create a credentials list we are very proud of.',
+    'map-spzoo': 'Sp. z o. o.',
+    'map-adres': 'ul. Chorzowska 58, 44-100 Gliwice',
+    'map-nip': 'VAT-EU:',
+    'map-nip-nr': 'PL6645344563',
+    'map-email': 'E-mail address:',
+    'map-tel': 'Phone no.:',
+    'map-social': 'Aurator in social media:'
+  }
+};
+
+function translateTexts(lang) {
+  $('.js-lang')
+    .toArray()
+    .forEach((element) => {
+      element.innerHTML = arrLang[lang][element.getAttribute('key')];
+    });
+}
+
+function setCurrentLanguage(lang) {
+  $('.js-translate')
+    .toArray()
+    .forEach((translateLink) => {
+      if (translateLink.getAttribute('key') === lang) {
+        translateLink.classList.add('h-current-lang');
+      } else {
+        translateLink.classList.remove('h-current-lang');
+      }
+    });
+
+  $('.c-lang__toggle').text(lang);
+  localStorage.setItem('currentLangAurator', lang);
+}
+
+function resetDropdownState() {
+  $('.c-lang__icon').removeClass('d-none');
+  $('.c-lang__icon-hide').addClass('d-none');
+  $('.c-lang__dd').addClass('d-none');
+}
+
+let currentLanguage;
+
+export function translateOnInit() {
```

Comment:
Could we move `currentLanguage` to `translateOnInit`?

- [ ] move currentLanguage to translateOnInit